queries/help/injections.scm

```diff
@@ -0,0 +1 @@
+(code_block) @lua
```

Comment:
Yes, that's the main question on this PR.

I used Lua because I knew @dundargoc would primarily look at that ;)

Also, the audience for nvim-treesitter is probably more interested in Lua than Vimscript, and I wanted to see if mis-highlighting the latter (or Python, or pseudocode, or...) would be worse than no highlighting at all.

Reply:
I think that mishighlighting is worst that now highlithing bacause it is misleading.

Reply:
we could also experiment with some kind of autodetect. Maybe when in doubt and first parser yields errors maybe the next parser can parse sucessfully.

Reply:
I'd start with some heuristics like vim uses for filetype disambiguation (look at the first line(s), see if there's something like `local` or `let`)?